docs/userGuide/usingPlugins.md

```diff
@@ -106,6 +108,31 @@ module.exports = {
 </div>
 ```
 
+#### Assets
+
+Plugins can call the methods `addLinks` and `addScripts` to add additional assets to the page. 

```

Comment:
- call → implement
- Since they return assets instead of directly add, how about:
  - `addLinks` → `getLinks`?
  - `addScripts` → `getScripts`?